samples/WorkerBindingSamples/Table/TableInputBindingSamples.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Text;
+using Azure.Data.Tables;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace WorkerBindingSamples.Table
+{
+    public class TableInputBindingSamples
```

Comment:
Did you test all of these scenarios with user and system managed identity?

Reply:
Yes tested in all scenarios!